chapters/publication.tex

```diff
@@ -1,96 +1,171 @@
-with which they are already comfortable,
-and since they have worked with many others in the past,
-that workflow is likely to be the least-common-denominator:
-Microsoft Word with tracked changes.
-This chapter will show you how you can avoid at least some
-of the pain of Microsoft Word,
-while still providing materials in the format
-that co-authors prefer and journals request.
+In this section we suggest several methods --
+collectively refered to as ``dynamic documents'' --
+for managing the process of collaboration on any technical product.
+
+For most research projects, completing a written piece is not the end of the task.
+In almost all cases, you will be required to release a replication package,
+which contains the code and materials needed to create the results.
+These represent an intellectual contribution in their own right,
+because they enable others to learn from your process
+and better understand the results you have obtained.
+Holding code and data to the same standards as written work
+is a new discipline for many researchers,
+and here we provide some basic guidelines and responsibilities for both
+that will help you prepare a functioning and informative replication package.
+In all cases, we note that technology is rapidly evolving
+and that the specific tools noted here may not remain cutting-edge,
+but the core principles involved in publication and transparency will endure.
 \end{fullwidth}
 
 %------------------------------------------------
 
-\section{Collaborating on academic writing}
+\section{Collaborating on technical writing}
+
+It is increasingly rare that a single author will prepare an entire manuscript alone.
+More often than not, documents will pass back and forth between several writers
+before they are ready for publication,
+so it is essential to use technology and workflows that avoid conflicts.
+Just as with the preparation of analytical outputs,
+this means adopting tools and practices that enable tasks
+such as version control and simultaneous contribution.
+Furthermore, it means preparing documents that are \textbf{dynamic} --
+meaning that updates to the analytical outputs that constitute them
+can be passed on to the final output with a single process,
+rather than copy-and-pasted or otherwise handled individually.
+Thinking of the writing process in this way
+is intended to improve organization and reduce error,
+such that there is no risk of materials being compiled
+with out-of-date results, or of completed work being lost or redundant.
+
+\subsection{Dynamic documents}
+
+Dynamic documents are a broad class of tools that enable such a workflow.
+The term ``dynamic'' can refer to any document-creation technology
+that allows the creation of explicit references to raw output files.
+This means that, whenever outputs are updated,
+the next iteration of the document will automatically include
+all changes made to all outputs without any additional intervention from the writer.
```

Comment:
"_writer_" - I think _author_ or _user_ is better than writer. It could be an RA not responsible for writing that is tasked to do things related to this

Reply:
de4dd2b9815c4b9d78acab05b72be2cdbceb2eeb